docs/guide/using-plugins.md

```diff
@@ -82,6 +82,14 @@ export default defineConfig({
 })
 ```
 
+A function can also be used for more precise control:
+
+```js
+apply(config, { command }) {
+  return command === 'build' && !config.build.ssr
+}
+```
```

Comment:
Giving this is not really a common use case, as api-plugin.md already covers it, I think we can omit it here.